model/src/main/java/com/epam/esm/dto/CertificateDTO.java

```diff
@@ -0,0 +1,131 @@
+package com.epam.esm.dto;
+
+import com.epam.esm.entity.Tag;
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public class CertificateDTO {
+
+    private int id;
+
+    @Size(min = 3, max = 20, message = "The certificate name could be between 3 and 20 symbols")
+    private String name;
+
+    @Size(min = 5, max = 100, message = "Description could be between 5 and 100 symbols")
+    private String description;
+
+    @DecimalMin(value = "0", message = "Enter certificate price")
+    private Integer price;
+
+    @DecimalMin(value = "1", message = "Enter certificate duration more than 1 day")
+    private int duration;
+
+    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
+    private LocalDateTime createDate;
```

Comment:
I suggest using Instant class, it will handle time correctly regardless of client timeZone.